packages/mobile/src/screens/home/components/lot-section/lot-section.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC } from 'react';
+import { View, Text, FlatList, TextButton } from '~/components/components';
+import { globalStyles } from '~/styles/styles';
+import { LotType } from '~/common/enums/enums';
+import { LotParams } from '~/common/types/types';
+import { FlexStyle } from 'react-native';
+import { Lot } from '../components';
+import { styles } from './styles';
+
+type Props = {
+  sectionTitle: string;
+  extendTitle: string;
+  lotType: LotType;
+  data: LotParams;
+  onExtendPress: () => void;
+  wrapperStyles?: FlexStyle[];
+};
+
+const LotSection: FC<Props> = ({
+  sectionTitle,
+  extendTitle,
+  lotType,
+  data,
+  onExtendPress,
+  wrapperStyles,
+}) => {
+  const dataLengthCheck = (data: LotParams) => {
```

Comment:
For which purpose that code part?

Reply:
This was in the task description :
**popular lots sections (auction type); up to 10 cards, see all button** 
This function cut the array length if there are more then 10 cards